src/workerd/api/crypto/rsa.c++

```diff
@@ -28,6 +30,32 @@ kj::Maybe<T> fromBignum(kj::ArrayPtr<kj::byte> value) {
 
   return asUnsigned;
 }
+
+kj::Array<kj::byte> bioToArray(BIO* bio) {
+  BUF_MEM* bptr;
+  BIO_get_mem_ptr(bio, &bptr);
+  auto buf = kj::heapArray<char>(bptr->length);
+  auto aptr = kj::arrayPtr(bptr->data, bptr->length);
+  buf.asPtr().copyFrom(aptr);
+  return buf.releaseAsBytes();
+}
+
+kj::Maybe<kj::Array<kj::byte>> simdutfBase64UrlDecode(kj::StringPtr input) {
+  auto size = simdutf::maximal_binary_length_from_base64(input.begin(), input.size());
+  auto buf = kj::heapArray<kj::byte>(size);
+  auto result = simdutf::base64_to_binary(input.begin(),
+                                          input.size(),
+                                          buf.asChars().begin(),
+                                          simdutf::base64_url);
+  if (result.error != simdutf::SUCCESS) return kj::none;
+  KJ_ASSERT(result.count <= size);
+  return buf.first(result.count).attach(kj::mv(buf));
+}
```

Comment:
Reviewer note: This is a more efficient alternative to the existing base64 url decoding method we have based on kj::decodeBase64. This one follows the standard forgiving base64 decode algorithm defined by whatwg specs. Soon I expect this to be moved out of this rsa.c++ file and into a shared location.